src/interests.rs

```diff
@@ -57,11 +60,17 @@ impl Interests {
         target_os = "ios",
         target_os = "macos"
     ))]
-    pub const AIO: Interests = Interests(unsafe { NonZeroU8::new_unchecked(AIO) });
+    pub const AIO: Interests = Interests(unsafe { NonZeroU16::new_unchecked(AIO) });
 
     /// Returns a `Interests` set representing LIO completion interests.
     #[cfg(target_os = "freebsd")]
-    pub const LIO: Interests = Interests(unsafe { NonZeroU8::new_unchecked(LIO) });
+    pub const LIO: Interests = Interests(unsafe { NonZeroU16::new_unchecked(LIO) });
+
+    /// TODO
+    pub const READ_CLOSE: Interests = Interests(unsafe { NonZeroU16::new_unchecked(READ_CLOSE) });
```

Comment:
I don't know if we want to do this, I believe that kqueue has this enabled (or tracks this) by default, we might just want to replicate that in epoll/IOCP as well.

Reply:
Such that `READABLE` interests also register read close interests.

Reply:
@carllerche and I briefly talked about this in Discord--short back and forth but basically was we may want to avoid registering interest in events that the user doesn't ask for.

I realize now that on `kqueue` platforms we may need to `cfg` these methods since this is tracked by default, in which case I may prefer having `EPOLLRDHUP` automatically register if `READABLE` interest is registered.